test/langtools/jdk/javadoc/doclet/testRecordTypes/TestRecordTypes.java

```diff
@@ -560,4 +558,51 @@ public void testDeprecatedRecordComponent(Path base) throws IOException {
                     <div class="col-last even-row-color"></div>
                     </div>""");
     }
+
+    @Test
+    public void testSerializableType(Path base) throws IOException {
+        Path src = base.resolve("src");
+        tb.writeJavaFiles(src,
+                """
+                    /**
+                     * A point,
+                     * @param x the x coord
+                     * @param y the y coord
+                     */
+                    public record Point(int x, int y) implements java.io.Serializable { }""");
+
```

Comment:
I just noted the static files in the `examples` directory already contain a `public record SerializablePoint(int x, int y) implements Serializable { }`, but it seems the output is never checked. Not an error, just a bit of tiny bit of wasted resources.

Reply:
Yeah. IIRC, the examples were specifically taken from public documentation so have a separate role.  I think the test is clearer as is.